packages/paste-website/src/context/DarkModeContext.tsx

```diff
@@ -0,0 +1,18 @@
+import * as React from 'react';
+import type {UseDarkModeReturn} from '../hooks/useDarkMode';
+
+type DarkModeContextValue = {
+  theme: UseDarkModeReturn[0];
+  toggleMode: UseDarkModeReturn[1];
+  componentMounted: UseDarkModeReturn[2];
```

Comment:
What does this do?

Reply:
Specifically `componentMounted` or Dark mode context?